### PR TITLE
handle case when noIndex is specified.

### PR DESCRIPTION
**Pull request by vezzi**
Number of reads is fetched from undetermined and used as PF clusters. A NotOrginal flag is added to the document

### CODE REVIEW

**Comment by vezzi** [comment]
@Galithil @sylvinite

File: taca/analysis/analysis.py [hunk 1]
```diff
@@ -86,6 +86,33 @@ def _upload_to_statusdb(run):
...
+            #in Lanes_stats fix the lane yield
+            parser.obj['illumina']['Demultiplex_Stats']['Lanes_stats'][int(lane) - 1]['PF Clusters'] = str(PFclusters)
+            #now fix Barcode lane stats
+            updated = 0 #check that only one update is made
+            for sample in parser.obj['illumina']['Demultiplex_Stats']['Barcode_lane_statistics']:
+                if lane in sample['Lane']:
+                    updated +=1
+                    sample['PF Clusters'] = str(PFclusters)
+            if updated != 1:
+                logger.error("While taking extra care of lane {} of NoIndex type I updated more than once the barcode_lane. This is too uc to contniue so I will fail.".format(lane))
```
Comment (Galithil) [inline taca/analysis/analysis.py]: typo : contniue

**Comment by Galithil** [comment]
okay, that looks good.
I don't mind typo in comments, but I mind typos in logs because I usually grep for them.

**Comment by vezzi** [comment]
@Galithil fixed

**Pull request merged**